data/presets/place/municipality.json

```diff
@@ -0,0 +1,17 @@
+{
+    "icon": "maki-triangle-stroked",
+    "fields": [
+        "name",
+        "population",
+        "population/date",
+        "source/population"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "place": "municipality"
+    },
+    "name": "Municipality"
```

Comment:
“Municipality” would cause a lot of confusion in the default American English locale. “Municipality” is the proper translation of something like a _municipio_ in various Spanish-speaking countries – a kind of “space-filling” administrative area that completely partitions a larger administrative area, not necessarily urban. But to an American English speaker, “municipality” is normally a catch-all term for an incorporated city, town, or village, corresponding to a populated place.

“Township” might be a better name for this preset. In the United States, we use `place=municipality` for civil townships in the Midwest and towns in New England, New York, and Wisconsin, all of which have the same semantics. But “township” means something completely different in other English-speaking countries like South Africa. Probably we can’t responsibly add this preset until we split it up into multiple presets tailored to each region’s legal constructs.

Reply:
Would a possible solution be to just split this preset into two _regional_ ones: One with `name=Township` for the US, and one with `name=Municipality` for everywhere else (or perhaps just the [countries](https://taginfo.openstreetmap.org/tags/place=municipality#map) like Italy, Germany, Sweden, Thailand(?) where it is commonly used in OSM).